### PR TITLE
Use templates insted of static html files

### PR DESCRIPTION
**Pull request by pafff**
@bitrut
https://gist.github.com/pafff/ea854993d8ff27e4352c04792c0a49b7

### CODE REVIEW

File: src/presence_analyzer/templates/base.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <div id="content">
+            <h2>{% block content_header %}Presence analyzer{% endblock %}</h2>
+            <p>
+                <select id="user_id" style="display: none">
+                    <option value="">--</option>
+                </select>
+                <div id="chart_div" style="display: none">
+                </div>
+                <div id="loading">
+                    <img src="/static/img/loading.gif" />
```
Comment (bitrut) [inline src/presence_analyzer/templates/base.html]: You forgot to use `url_for`.

**Comment by bitrut** [comment]
You should avoid writing inline JS scripts and CSS styling (inside HTML document).
Please see: http://sekurak.pl/czym-jest-content-security-policy/

**Comment by bitrut** [comment]
Please mention reviewer by `@` when you finish suggested fixes.
Code looks fine :+1: 
Please squash commits.

**Comment by pafff** [comment]
Squashed commits
@bitrut

**Comment by bitrut** [comment]
Great, you can merge PR as I don't have write access to this repository.

**Pull request merged**